lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -236,23 +236,41 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
     'random' => TRUE,
   ];
 
+  // If the Rogue feature flag is on, grab reportbacks from Rogue.
+  if (variable_get('rogue_collection', FALSE)) {
+    // Hit the GET /reportbacks endpoint in Rogue with campaign_id filter.
+    // On the Rogue side, it will only return 'approved', which will be shown in the gallery.
+    $promotedReportbackItems = dosomething_rogue_get_reportback_items(['campaign_id' => $gallery['campaign_id']])['data'];
```

Comment:
This only returns 20 at a time since this endpoint is paginated. I was thinking to check in the response if ``links`['next']` is not null in the `meta` tag, hit the next page and add to `$promotedReportbackItems` but I don't think this is the best way to handle this logic. Any suggestions @DFurnes @sbsmith86 @katiecrane ?